src/Ocelot/Tracing/IOcelotRequestTracer.cs

```diff
@@ -0,0 +1,17 @@
+﻿using Butterfly.OpenTracing;
+using Microsoft.AspNetCore.Http;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Ocelot.Tracing
+{
+    public interface IOcelotRequestTracer
+    {
+        ISpan OnBeginRequest(HttpContext httpContext, Ocelot.Request.Request request);
+
+        void OnEndRequest(HttpContext httpContext, Ocelot.Request.Request request);
+
+        //void OnException(HttpContext httpContext, Exception exception, string @event);
```

Comment:
Do we need this? If yes we should plug it in, if not we should delete this.

Reply:
We need to bury those data points.

Reply:
we should delete this